packages/components/pages/f-checkout/src/components/CheckoutForm.vue

```diff
@@ -171,9 +173,13 @@ export default {
          */
         scrollToFirstInlineError () {
             this.$nextTick(() => {
-                const firstInlineError = document.querySelector('[data-js-error-message]');
+                const firstInlineError = document.querySelector('[aria-invalid="true"]');
 
                 this.scrollToElement(firstInlineError, { offset: -100 });
+
+                firstInlineError?.focus({
```

Comment:
This might not be a good idea.  Repositioning the cursor is helpful for a signed person, but for a blind person is frustrating.
It's a bit like moving somebody in a wheelchair without asking them if they want to be moved.

Reply:
Ah ok cool. The reason I tried it is because the screen already moves up to be on that field and it reads that particular error so thought it would help 😂 
Happy to remove it though